tests/models/test_decoders.py

```diff
@@ -311,12 +319,37 @@ def __maybe_reset_model(model, is_gptq):
                 res /= 20.0
             param.copy_(res)
 
+global global_model
+global_model = None
+def get_global_model_or_create(is_gptq, **kwargs):
```

Comment:
I have a feeling there must be a better way to share an object between all tests in pytest already... but I'm not an expert in pytest